runtime/doc/options.txt

```diff
@@ -3255,8 +3255,8 @@ A jump table for the options with a short description can be found at |Q_op|.
 			*'ignorecase'* *'ic'* *'noignorecase'* *'noic'*
 'ignorecase' 'ic'	boolean	(default off)
 			global
-	Ignore case in search patterns, completion, and when searching the tags file.
-	See also 'smartcase' and 'tagcase'.
```

Comment:
Did you make a mistake here @zeertzjq ? 

 I don't think the line "See also 'smartcase' and 'tagcase'." should have been related to this change? I don't believe I removed that in my Vim commit.

Reply:
Hmm I'll take a look later.